server/tests/db/test_01_es6.js

```diff
@@ -327,6 +338,132 @@ tests.NumberMaxSafeInteger = function() {
       'Number.isSafeInteger(Number.MAX_SAFE_INTEGER + 1);');
 };
 
+tests.NumberMinSafeInteger = function() {
+  console.assert(Number.MIN_SAFE_INTEGER - 1 === -Math.pow(2, 53),
+      'Number.MIN_SAFE_INTEGER');
+  console.assert(Number.isSafeInteger(Number.MIN_SAFE_INTEGER),
+      'Number.isSafeInteger(Number.MIN_SAFE_INTEGER);');
+  console.assert(!Number.isSafeInteger(Number.MIN_SAFE_INTEGER - 1),
+      'Number.isSafeInteger(Number.MIN_SAFE_INTEGER - 1);');
+};
+
+///////////////////////////////////////////////////////////////////////////////
+// Math
+
+tests.MathAcosh = function() {
+  console.assert(Number.isNaN(Math.acosh(-1)), 'Math.acosh -1');
+  console.assert(Number.isNaN(Math.acosh(0)), 'Math.acosh 0');
+  console.assert(Number.isNaN(Math.acosh(0.5)), 'Math.acosh 0.5');
+  console.assert(Math.acosh(1) === 0, 'Math.acosh 1');
+  console.assert(Math.acosh(2).toPrecision(10) === '1.316957897', 'Math.acosh 2');
+};
+
+tests.MathAsinh = function() {
+  console.assert(Math.asinh(1).toPrecision(10) === '0.8813735870', 'Math.asinh 1');
+  console.assert(Math.asinh(0) === 0, 'Math.asinh 0');
+};
+
+tests.MathAtanh = function() {
+  console.assert(Number.isNaN(Math.atanh(-2)), 'Math.atanh -2');
+  console.assert(Math.atanh(-1) === -Infinity, 'Math.atanh -1');
+  console.assert(Math.atanh(0) === 0, 'Math.atanh 0');
+  console.assert(Math.atanh(0.5).toPrecision(10) === '0.5493061443', 'Math.atanh 0.5');
+  console.assert(Math.atanh(1) === Infinity, 'Math.atanh 1');
+  console.assert(Number.isNaN(Math.atanh(2)), 'Math.atanh 2');
+};
+
+tests.MathCbrt = function() {
+  console.assert(Number.isNaN(Math.cbrt(NaN)), 'Math.cbrt NaN');
+  console.assert(Math.cbrt(-1) === -1, 'Math.cbrt -1');
+  console.assert(Object.is(Math.cbrt(-0), -0), 'Math.cbrt -0');
+  console.assert(Math.cbrt(-Infinity) === -Infinity, 'Math.cbrt -Infinity');
+  console.assert(Object.is(Math.cbrt(0), 0), 'Math.cbrt 0');
+  console.assert(Math.cbrt(1) === 1, 'Math.cbrt 1');
+  console.assert(Math.cbrt(Infinity) === Infinity, 'Math.cbrt Infinity');
+  console.assert(Math.cbrt(null) === 0, 'Math.cbrt null');
+  console.assert(Math.cbrt(2).toPrecision(10) === '1.259921050', 'Math.cbrt 2');
```

Comment:
Maybe start with a really trivial case, like `Math.cbrt(27) === 3` or the like, in part just to make it obvious what the "cbrt" function is.